src/nvim/ui.coffee

```diff
@@ -8,6 +8,43 @@ remote = require 'remote'
 config = require './config'
 {keystrokeForKeyboardEvent} = require './key_handler'
 
+# borrowed from underscore@acf826fa42ba6
+_ = {}
+
+_.now = Date.now or ->
+  (new Date).getTime()
+
+_.debounce = (func, wait, immediate) ->
```

Comment:
The whole `_` thing is still too much, `debounce` looks like a function wrapper, yet it's called only once, so you can directly wrap that function only